skins/laika/src/components/pages/donation_form/subpages/AddressPage.vue

```diff
@@ -95,8 +107,17 @@ export default Vue.extend( {
 				return this.$store.getters[ NS_PAYMENT + '/isBankTransferPayment' ];
 			},
 		},
+		paymentWasInitialized: {
+			get(): boolean {
+				return this.$store.state[ NS_PAYMENT ].initialized;
```

Comment:
I am ok with this way of writing the computed property but if we look at what we did in other components (maybe in the later stage of the development) that would be a vuex getter and here we'll just use the spread operator.